javascript/index.js

```diff
@@ -0,0 +1,6 @@
+const getCardAnswer = document.querySelector('[data-js="card__answer"]');
+const showAnswer = document.querySelector('[data-js="card__answer-button"]');
+
+showAnswer.addEventListener("click", () => {
+  getCardAnswer.classList.toggle("card__answer--visible");
+});
```

Comment:
You did not add an HTML element with the class `class__answer--visible`, so this does not do anything right now, does it?

Reply:
It only toggles the color on click. I haven't figured out how to hide and unhide the answer but I still wanted to make sure the button works when it's clicked on.

Reply:
Hint: on click, add a class "currywurst"; in your .css file, pass `display: none` to the class "currywurst".